typescript/mocks/readTypeScriptModules/inheritedMembers.ts

```diff
@@ -0,0 +1,12 @@
+export class LastParent {
+  lastParentProp: number = 0;
+}
+
+export class FirstParent extends LastParent {
+  firstParentProp: string = 'Works';
+  _privateProperty: string = 'Private';
```

Comment:
you can also specify privacy by putting the `private` keyword before the property